src/index.html

```diff
@@ -8,6 +8,39 @@
+      </symbol>
+      <symbol id="icon-share" viewBox="0 0 32 32">
+      <title>share</title>
+      <path d="M16 10c-1.5 0-2.8 0.5-3.8 1.4 0.4-0.2 0.8-0.4 1.3-0.4 1.4 0 2.5 1.1 2.5 2.5s-1.1 2.5-2.5 2.5-2.5-1.1-2.5-2.5c0-0.5 0.1-1 0.4-1.3-0.9 1-1.4 2.4-1.4 3.8 0 3.3 2.7 6 6 6s6-2.7 6-6-2.7-6-6-6z"></path>
+      <path d="M16 6c-9 0-15 8.9-15 10s6 10 15 10 15-8.9 15-10c0-1.1-6-10-15-10zM16 24c-4.4 0-8-3.6-8-8s3.6-8 8-8 8 3.6 8 8-3.6 8-8 8z"></path>
+      </symbol>
+      <symbol id="icon-publish" viewBox="0 0 32 32">
+      <title>publish</title>
+      <path d="M28 12.8v12.2c0 0.6-0.4 1-1 1h-23c-0.6 0-1-0.4-1-1v-17c0-0.6 0.4-1 1-1h18.2l-2-2h-16.2c-1.7 0-3 1.3-3 3v17c0 1.7 1.3 3 3 3h23c1.7 0 3-1.3 3-3v-10.2l-2-2z"></path>
+      <path d="M23.2 4c-0.4 0-0.7 0.5-0.4 0.9l2.2 2.2-5.1 5.1c-0.2 0.2-0.2 0.5 0 0.7l2.1 2.1c0.2 0.2 0.5 0.2 0.7 0l5.1-5.1 2.2 2.2c0.3 0.3 0.9 0.1 0.9-0.4v-7.7h-7.7z"></path>
+      </symbol>
+      <symbol id="icon-up" viewBox="0 0 32 32">
+      <title>up</title>
+      <path d="M28 6c0.6 0 1 0.4 1 1v18c0 0.6-0.4 1-1 1h-24c-0.6 0-1-0.4-1-1v-18c0-0.6 0.4-1 1-1h24zM28 4h-24c-1.7 0-3 1.3-3 3v18c0 1.7 1.3 3 3 3h24c1.7 0 3-1.3 3-3v-18c0-1.7-1.3-3-3-3v0z"></path>
+      </symbol>
+      <symbol id="icon-up1" viewBox="0 0 32 32">
+      <title>up1</title>
+      <path d="M28 4h-24c-1.7 0-3 1.3-3 3v18c0 1.7 1.3 3 3 3h24c1.7 0 3-1.3 3-3v-18c0-1.7-1.3-3-3-3zM29 7v8h-9v-1c0-1.1-0.9-2-2-2h-1v-6h11c0.6 0 1 0.4 1 1zM4 6h11v6h-3v3h-9v-8c0-0.6 0.4-1 1-1zM3 25v-8h9v1c0 1.1 0.9 2 2 2h1v6h-11c-0.6 0-1-0.4-1-1zM28 26h-11v-6h1c1.1 0 2-0.9 2-2v-1h9v8c0 0.6-0.4 1-1 1z"></path>
+      </symbol>
+      <symbol id="icon-up2" viewBox="0 0 32 32">
+      <title>up2</title>
+      <path d="M28 28h-2v-2h2v2zM24.4 28h-2v-2h2v2zM20.8 28h-0.8c-0.6 0-1.2-0.2-1.7-0.5l1.1-1.6c0.2 0.1 0.4 0.2 0.6 0.2h0.8v1.9zM30.1 27.1l-1.4-1.4c0.2-0.2 0.3-0.4 0.3-0.7v-0.5h2v0.5c0 0.8-0.3 1.6-0.9 2.1zM17.1 25.6c0-0.2-0.1-0.4-0.1-0.6v-1.7h2v1.7c0 0.1 0 0.1 0 0.2l-1.9 0.4zM31 23h-2v-2h2v2zM19 21.8h-2v-2h2v2zM31 19.4h-2v-2h2v2zM19 18.2h-2v-2h2v2zM31 15.8h-2v-2h2v2zM19 14.6h-2v-2h2v2zM31 12.2h-2v-2h2v2zM19 11h-2v-2h2v2zM31 8.7h-2v-1.7c0-0.1 0-0.1 0-0.2l2-0.4c0 0.2 0 0.4 0 0.6v1.7zM19 7.5h-2v-0.5c0-0.8 0.3-1.6 0.9-2.1l1.4 1.4c-0.2 0.2-0.3 0.4-0.3 0.7v0.5zM28.6 6.2c-0.2-0.1-0.4-0.2-0.6-0.2h-0.8v-2h0.8c0.6 0 1.2 0.2 1.7 0.5l-1.1 1.7zM20 6v-1l-0.2-1h2.2v2h-2zM25.6 6h-2v-2h2v2z"></path>
+      <path d="M12 6c0.6 0 1 0.4 1 1v18c0 0.6-0.4 1-1 1h-8c-0.6 0-1-0.4-1-1v-18c0-0.6 0.4-1 1-1h8zM12 4h-8c-1.7 0-3 1.3-3 3v18c0 1.7 1.3 3 3 3h8c1.7 0 3-1.3 3-3v-18c0-1.7-1.3-3-3-3v0z"></path>
+      </symbol>
+      </defs>
+    </svg>
```

Comment:
Having these icons in the `index.html` seems odd. Would it be possible to have them in a separate file and loaded by Webpack?

Reply:
Yeah, its weird.  I never used svg icons like this but this is the recommended way in the icomoon documentation.